src/central-server/admin-service/int-test/src/intTest/resources/application-override.yml

```diff
@@ -25,6 +25,7 @@ test-automation:
     testable-container:
       reuse-between-runs: ${reuse-between-runs}
       spring-boot:
+        base-image: azul/zulu-openjdk:17
```

Comment:
Could we use the default JAVA package available under Ubuntu 20/22 or RHEL 8 instead of Azul? Or as an alternative use `temurin` if we decide to suggest it for RHEL7 as well.

Reply:
Using eclipse-temurin:17-jammy as base image in int tests.